optimum/habana/transformers/generation/utils.py

```diff
@@ -76,6 +76,12 @@
 
 logger = logging.get_logger(__name__)
 
+def unwrap_ds(model):
+    if hasattr(model, 'module'):
+        return model.module
+    return model
```

Comment:
I would prefer to have this method defined here: https://github.com/huggingface/optimum-habana/blob/main/optimum/habana/transformers/deepspeed.py
since it is related to DeepSpeed
Let's also rename it to `unwrap_deepspeed_model` so that the name is clearer.